controllers/componentversion_controller.go

```diff
@@ -52,7 +52,7 @@ type ComponentVersionReconciler struct {
 //+kubebuilder:rbac:groups="",resources=services;pods,verbs=get;create;update;patch;delete
 //+kubebuilder:rbac:groups="apps",resources=deployments,verbs=get;create;update;patch;delete
 
-// +kubebuilder:rbac:groups="",resources=secrets;serviceaccounts,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=secrets;serviceaccounts,verbs=create;get;list;watch
```

Comment:
the component version controller shouldn't need to create a service account, should it?

Reply:
I think we can remove the create here as it doesn't need to create secrets or service accounts.

Reply:
Yeah. I added create for the `secret` a commit ago when we were creating the cert secret. But now, we just expect it to be there. So yeah, I can remove the `create` here.

Reply:
```suggestion
// +kubebuilder:rbac:groups="",resources=secrets;serviceaccounts,verbs=get;list;watch
```

Reply:
Yah, I still have to do this. :D I won't forget. Doing it now.